helm/charts/determined/templates/master-config.yaml

```diff
@@ -106,6 +115,25 @@ stringData:
         password: {{ required "A valid password type is required!" .Values.scim.auth.password }}
         {{- end }}
     {{- end }}
+
+    {{- if .Values.saml }}
+    saml:
+      enabled: {{ .Values.saml.enabled | default false }}
+      provider: {{ required "A valid provider entry is required!" .Values.saml.provider}}
+      idp_recipient_url: {{ required "A valid recipient url is required!" .Values.saml.idpRecipientUrl }}
+      idp_sso_url: {{ required "A valid sso url is required!" .Values.saml.idpSsoUrl }}
+      idp_sso_descriptor_url: {{ required "A valid sso descriptor url is required!" .Values.saml.idpSsoDescriptorUrl }}
+      idp_cert_path: {{ required "A valid idp cert path is required!" .Values.saml.idpCertPath }}
+      {{- if .Values.saml.autoProvisionUsers }}
+      auto_provision_users: {{ .Values.saml.autoProvisionUsers }}
+      {{- end }}
+      {{- if .Values.saml.groupsAttributeName }}
+      groups_claim_name: {{ .Values.saml.groupsAttributeName }}
+      {{- end }}
+      {{- if .Values.saml.displayNameAttributeName }}
+      display_name_attribute_name: {{ .Values.saml.displayNameAttributeName }}
```

Comment:
Maybe it is better to make it `display_name_claim_name` if we are changing `groups_attribute_name` to `groups_claim_name`

Same change in values.yaml too